src/DOMTokenList.js

```diff
@@ -23,7 +23,10 @@ module.exports = class DOMTokenList extends Array {
   }
 
   contains(token) {
-    return this.includes(token);
+    for (let i = 0, n = this.length; i < n; i++) {
```

Comment:
and here you are avoiding native includes ... I'm not too fund of augmenting complexity for no reason when the platform provides default alternatives